wagtail/wagtailcore/tests/test_widgets.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from wagtail.wagtailcore.fields import BaseTextAreaWidget, RichTextArea
+
+
+class TestBaseTextAreaWidget(TestCase):
+
+    def test_get_panel_implemented(self):
+        with self.assertRaises(NotImplementedError):
+            widget = BaseTextAreaWidget()
+            widget.get_panel()
+
+    def test_render_js_init(self):
```

Comment:
I think we can remove this test because it checks `WidgetWithScript.render_js_init`. Probably `WidgetWithScript` should have own tests.